platform/broadcom/sonic-platform-modules-ruijie/b6510-48vs8cq/sonic_platform/chassis.py

```diff
@@ -417,7 +417,7 @@ def set_status_led(self, color):
         if regval is None:
             print("Invaild color input.")
             return False
-        ret , log = subprocess.getstatusoutput(self.set_sys_led_cmd + regval)
+        ret , log = getstatusoutput_noshell(self.set_sys_led_cmd.append(regval))
```

Comment:
>append [](http://example.com/codeflow?start=65&length=6)

It will change variable, so next call will be unexpected.

Reply:
You initialize the member var `self.set_sys_led_cmd` but actually not using it.

Suggest
```
cmd = self.set_sys_led_cmd + [regval]
getstatusoutput_noshell(cmd)
```